java/src/main/c++/vowpalWabbit_learner_VWMulticlassLearner.cc

```diff
@@ -11,3 +11,17 @@ JNIEXPORT jint JNICALL Java_vowpalWabbit_learner_VWMulticlassLearner_predict(JNI
 JNIEXPORT jint JNICALL Java_vowpalWabbit_learner_VWMulticlassLearner_predictMultiline(JNIEnv *env, jobject obj, jobjectArray example_strings, jboolean learn, jlong vwPtr)
 { return base_predict<jint>(env, example_strings, learn, vwPtr, multiclass_predictor);
 }
+
+jfloatArray multiclass_raw_predictor(example* vec, JNIEnv *env){
+  size_t num_values = vec->l.cs.costs.size();
+  jfloatArray j_labels = env->NewFloatArray(num_values);
+  for (int i=0 ; i<num_values; i++) {
+    jfloat f[] = { vec->l.cs.costs[i].partial_prediction };
+    env->SetFloatArrayRegion(j_labels, i, 1, (float*)f);
```

Comment:
Expected a double here instead of an array because there should be one value. I ran tests to verify array sizes are all 1 in this case. My understanding is there could be cases where array sizes can be >1 (adf?). I can investigate further if we decide to go with the PR.